pelican/tools/pelican_quickstart.py

```diff
@@ -12,6 +12,11 @@
 import pytz
 
 try:
+    import readline  #NOQA
```

Comment:
Adding a space between `#` and `NOQA` will fix `flake8`'s complaint about the unused import.